.gitlab-ci.yml

```diff
@@ -14,6 +14,7 @@ variables:
 .runtests:
   before_script:
     - "conda create --prefix /root/sklldev --channel conda-forge --file requirements.dev python=${PYVERSION} curl --yes --quiet"
+    - /root/sklldev/bin/pip install wandb
```

Comment:
Looks like conda-forge folks are working on releasing 0.17.4: https://github.com/conda-forge/wandb-feedstock/pull/106

Reply:
Great, Should we wait until the PR gets merged?

Reply:
@desilinguist  wandb just released a version that supports Numpy 2. This PR is ready.